Test/exceptions/Exceptions1.dfy

```diff
@@ -0,0 +1,79 @@
+// RUN: %dafny /compile:3 "%s" > "%t"
```

Comment:
I suggest including a `/dprint` or `/rprint` argument as well, naming a new temporary file in the Output folder (see some other test files). This will test whether or not printing crashes on this input. (Note, I'm not suggesting comparing the printed file, just producing it.)